tests/test_stories.py

```diff
@@ -9,57 +9,141 @@
+        response = r.json()
+        story_with_pois['poi_ids'].append(response['result']['poi']['_id'])
+        os.environ['POI_ID_2'] = str(response['result']['poi']['_id'])
+        r = requests.post('http://127.0.0.1:5000/stories', json=story_with_pois)
+        response = r.json()
+        self.assertEqual(response['code'], 201)
+        self.assertEqual(response['result']['story']['story_name'], story_with_pois['story_name'])
+        os.environ['STORY_ID_2'] = str(response['result']['story']['_id'])
+
+    def test3_get_stories(self):
+        # GET with no parameters
+        story_id_1 = int(os.environ.get('STORY_ID_1'))
+        story_id_2 = int(os.environ.get('STORY_ID_2'))
+        r = requests.get('http://127.0.0.1:5000/stories')
+        response = r.json()
+        self.assertEqual(response['code'], 200)
+        story = next((s for s in response['result']['stories'] if s['_id'] == story_id_1), None)
+        self.assertEqual(story['story_name'], story_with_no_pois['story_name'])
+        story = next((s for s in response['result']['stories'] if s['_id'] == story_id_2), None)
+        self.assertEqual(story['story_name'], story_with_pois['story_name'])
+
+    def test4_get_stories(self):
+        # GET with poi_id parameter
+        story_id_2 = int(os.environ.get('STORY_ID_2'))
+        poi_id_1 = int(os.environ.get('POI_ID_1'))
+        poi_id_2 = int(os.environ.get('POI_ID_2'))
+        r = requests.get('http://127.0.0.1:5000/stories?poi_id={}'.format(poi_id_1))
+        response = r.json()
+        self.assertEqual(response['code'], 200)
+        story = next((s for s in response['result']['stories'] if s['_id'] == story_id_2), None)
+        self.assertEqual(story['story_name'], story_with_pois['story_name'])
+        r = requests.get('http://127.0.0.1:5000/stories?poi_id={}'.format(poi_id_2))
+        response = r.json()
+        self.assertEqual(response['code'], 200)
+        story = next((s for s in response['result']['stories'] if s['_id'] == story_id_2), None)
+        self.assertEqual(story['story_name'], story_with_pois['story_name'])
+
+    def test5_put_stories(self):
+        # PUT with new story name
+        story_id_1 = int(os.environ.get('STORY_ID_1'))
+        r = requests.put('http://127.0.0.1:5000/stories/{}'.format(story_id_1), json=story_edit)
+        response_put = r.json()
+        self.assertEqual(response_put['code'], 200)
+        r = requests.get('http://127.0.0.1:5000/stories')
+        response_get = r.json()
+        story = next((s for s in response_get['result']['stories'] if s['_id'] == story_id_1), None)
+        self.assertEqual(story['story_name'], story_edit['story_name'])
+
+    def test6_put_stories(self):
+        # PUT with blank payload
+        story_id_1 = int(os.environ.get('STORY_ID_1'))
+        r = requests.put('http://127.0.0.1:5000/stories/{}'.format(story_id_1), json=story_empty)
+        response_put = r.json()
+        self.assertEqual(response_put['code'], 200)

```

Comment:
doing a PUT request with an "empty" story isn't valid and should return a `422`

Reply:
the purpose of this test should be to make sure that giving an invalid payload returns an error